### PR TITLE
fix: [CDS-43409]: include time tracking to extract fields

### PR DESCRIPTION
**Pull request by rafaelmart1ns**
## Checklist
- [ ] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>
  
- Feature build: `trigger feature-build`
- Immutable delegate `trigger publish-delegate`
</details>

<details>
  <summary>PR Check triggers</summary>
  
  You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ti0, ti1`
  
- Compile: `trigger compile`
- CodeFormat: `trigger codeformat`
- MessageMetadata: `trigger messagecheck`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- runDockerizationCheck: `trigger dockerizationcheck`
- runAuthorCheck: `trigger authorcheck`
- Checkstyle: `trigger checkstyle`
- PMD: `trigger pmd`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/37763)

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Failed_** 
 


            
 #### 4 Issues 
* Code issue 4 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/7752dbb3099d2dba3b5d37f5931a5e4c/pull-request/37763)

**Review by Fernando-Dourado — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/37763)* status: 2 file(s) to review. 1 change requests. 0 of 1 approvals obtained. 

___
*[`260-delegate/src/test/java/software/wings/delegatetasks/jira/JiraTaskTest.java` line 957 at r1](https://reviewable.io/reviews/harness/harness-core/37763#-NCWm0xC4OyWTZjlqOKn:-NCWm0xC4OyWTZjlqOKo:b-n34nkz) ([raw file](https://github.com/harness/harness-core/blob/a02d3c5fe7ae7831a196f26721b14be51d577596/260-delegate/src/test/java/software/wings/delegatetasks/jira/JiraTaskTest.java#L957)):*
> ```Java
>   public void shouldReturnSuccessfulExecutionForCheckJiraApproval() throws JiraException {
>     JiraTaskParameters taskParameters = getTaskParams(JiraAction.CHECK_APPROVAL);
>     ;
> ```

Please, remove this line

**Review by rafaelmart1ns — COMMENTED** [review]
> Reviewed 1 of 2 files at r1, 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/37763)* status: 2 file(s) to review. 1 change requests. 0 of 1 approvals obtained.

**Review by rafaelmart1ns — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/37763)* status: 2 file(s) to review. 1 change requests. 1. unresolved discussions. 0 of 1 approvals obtained. 

___
*[`260-delegate/src/test/java/software/wings/delegatetasks/jira/JiraTaskTest.java` line 957 at r1](https://reviewable.io/reviews/harness/harness-core/37763#-NCWm0xC4OyWTZjlqOKn:-NCWmv2t-UGUo2AMsHgI:b-7rl0jd) ([raw file](https://github.com/harness/harness-core/blob/a02d3c5fe7ae7831a196f26721b14be51d577596/260-delegate/src/test/java/software/wings/delegatetasks/jira/JiraTaskTest.java#L957)):*
<details><summary><i>Previously, Fernando-Dourado (Fernando Dourado) wrote…</i></summary><blockquote>

Please, remove this line
</blockquote></details>

done.

**Review by Fernando-Dourado — APPROVED** [review]
> Reviewed 1 of 2 files at r1, 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/37763)* status: 1. unresolved discussions.

**Review by Fernando-Dourado — APPROVED** [review]
> Reviewed 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/37763)* status: :shipit: complete! ✓

File: 260-delegate/src/test/java/software/wings/delegatetasks/jira/JiraTaskTest.java [hunk 1]
```diff
@@ -932,6 +954,7 @@ public void shouldReturnSuccessfulExecutionForGetCreateMetadata()
   @Category(UnitTests.class)
   public void shouldReturnSuccessfulExecutionForCheckJiraApproval() throws JiraException {
     JiraTaskParameters taskParameters = getTaskParams(JiraAction.CHECK_APPROVAL);
+
```
Comment (vcalasansh) [inline 260-delegate/src/test/java/software/wings/delegatetasks/jira/JiraTaskTest.java:957]: extra line

Comment (rafaelmart1ns) [reply 260-delegate/src/test/java/software/wings/delegatetasks/jira/JiraTaskTest.java:957]: added by format code.

**Review by vcalasansh — COMMENTED** [review]
> Reviewed 1 of 1 files at r2, 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/37763)* status: 1. unresolved discussions.

**Review by vcalasansh — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/37763#-:-NC_66Nk0WnPWixQeDzj:bnfp4nl)* status: 1. unresolved discussions.

**Review by rafaelmart1ns — COMMENTED** [review]
> Reviewed all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/37763)* status: 1. unresolved discussions.

**Review by rafaelmart1ns — COMMENTED** [review]
> Reviewed 1 of 1 files at r3.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/37763)* status: 1. unresolved discussions.

**Review by vcalasansh — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/37763)* status: :shipit: complete! ✓

**Pull request merged**